openpype/hosts/zbrush/api/launch_script.py

```diff
@@ -0,0 +1,85 @@
+import os
+import sys
+import signal
+import traceback
+import ctypes
+import platform
+import logging
+
+from qtpy import QtWidgets, QtCore, QtGui
+
+from openpype import style
+from openpype.pipeline import install_host
+from openpype.hosts.zbrush.api import (
+    ZbrushHost,
+    CommunicationWrapper,
+)
+
+log = logging.getLogger(__name__)
+
+
+def safe_excepthook(*args):
+    traceback.print_exception(*args)
+
+
+def main(launch_args):
+    # Be sure server won't crash at any moment but just print traceback
+    sys.excepthook = safe_excepthook
+
+    # Create QtApplication for tools
+    # - QApplicaiton is also main thread/event loop of the server
+    qt_app = QtWidgets.QApplication([])
+
+    tvpaint_host = ZbrushHost()
```

Comment:
`tvpaint_host` -> `zbrush_host`

Or if we want to share code more between hosts name it `application_host` or `dcc_host` or whatever.

Reply:
actually they are from the different host. so this needs to be changed. The websocket code and the launch script gets inspiration from other hosts.